tests/TestCase/Command/TemplateCommandTest.php

```diff
@@ -443,6 +443,11 @@ public function testGetContentWithRoutingPrefix()
      */
     public function testBakeView()
     {
+        if (version_compare(Configure::version(), '4.5.0', '<')) {
```

Comment:
You'll have to removed this check after merging as Bake 3 isn't going to be used for 4.x :)